src/redturtle/prenotazioni/browser/utilities/notify_upcomming_bookings.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from datetime import timedelta
+from logging import getLogger
+
+from plone import api
+from Products.Five.browser import BrowserView
+from zope.event import notify
+
+from redturtle.prenotazioni.events import BookingReminderEvent
+
+logger = getLogger(__name__)
+
+
+class NotifyUpcommingBookings(BrowserView):
+    """View to notify about the upcomming bookings"""
+
+    def __call__(self):
+        today = datetime.now().replace(hour=0, minute=0)
+        unrestricted_search = api.portal.get_tool(
+            "portal_catalog"
+        ).unrestrictedSearchResults
+
+        for brain in unrestricted_search(portal_type="PrenotazioniFolder"):
+            booking_folder = brain.getObject()
+            reminder_notification_gap = getattr(
+                booking_folder, "reminder_notification_gap", None
+            )
+
+            if not reminder_notification_gap:
+                logger.info(
```

Comment:
do not fill the log with not necessary logs.

Reply:
Why is no necessary?

Reply:
Because every day you will add a log entry for a booking folder that don't have that field compiled.
It could be that we don't want to setup reminders for that folder and we are filling logs with not needed infos.

The same as when you send messages.

The [better] alternative is to set a custom log file and write all the informations there. @mamico?